guide-app/src/components/shared/CalendarDatePicker.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+import React from "react";
+import { StyleSheet, Text, TouchableOpacity, View } from "react-native";
+import Icon from "react-native-vector-icons/MaterialIcons";
+
+import { Colors, TextStyles } from "@assets/styles";
+import { StyleSheetUtils } from "@utils";
+import { DateUtils } from "@utils";
+import { isToday } from "date-fns";
+
+const styles = StyleSheet.create({
+  datePickerContainer: {
+    flexDirection: "row",
+    justifyContent: "space-around",
+    alignItems: "center",
+    alignSelf: "center",
+    marginTop: 4,
+    marginBottom: 12,
+    width: "70%"
+  },
+  datePickerText: StyleSheetUtils.flatten([
+    TextStyles.defaultFontFamily,
+    {
+      fontSize: 18,
+      fontWeight: "500",
+      fontStyle: "normal",
+      color: Colors.black26
+    }
+  ])
+});
+
+type Props = {
+  chosenDate: Date,
+  currentLanguage: string,
+  // TODO: :)
```

Comment:
TODO
Please document what to do if leaving it in.

Reply:
Fixeed